packages/ListBox/src/components/Trigger/Trigger.js

```diff
@@ -92,11 +92,23 @@ export default function Trigger(props) {
   const [triggerButtonId] = React.useState(() => `list-box-trigger_${uuidv4()}`);
 
   const handleClick = () => {
-    if (isDisabled) {
+    if (state.isOpen) {
+      dispatch({ type: useListBox.types.closePopover });
```

Comment:
I don't really see much affect in the UI with this approach, though the code _is_ arguably clearer. 

This issue isn't really one with the ListBox though, it's present in the Popover too, and noted in the code comments:

https://github.com/acl-services/paprika/blob/master/packages/Popover/src/components/Content/Content.js#L49-L59

Apart from writing tests differently, the interaction seems improved in my experiments by increasing that timeout (I found 350ms was an improvement):

https://github.com/acl-services/paprika/blob/master/packages/Popover/src/components/Content/Content.js#L67